README.markdown

```diff
@@ -124,6 +127,12 @@ foo :: Int
 foo = 42
 ```
 
+Note that you should not place setup code inbetween the module header (`module
```

Comment:
Could the reason be that in Haskell `import` statements have to be on top of the file?  Have you tried putting setup code here that only imports a file?
(Just my wild guessing.)

Reply:
I can't understand what you mean by "putting setup code here that only imports a file"?

Reply:
Sorry, that was a wrong guess on my side.